src/Workspaces/Core/Portable/Formatting/Rules/BaseIndentationFormattingRule.cs

```diff
@@ -187,7 +187,7 @@ private static TextSpan GetSpanFromTokens(TextSpan span, SyntaxToken token1, Syn
             // current token is inside of the given span, get next token's start position.
             if (token2.Span.End <= span.End)
             {
-                token2 = token2.GetNextToken();
+                token2 = token2.GetNextToken(includeZeroWidth: true);
```

Comment:
Why is this needed on GetNextToken() here, but not on GetPreviousToken() above? If the difference is intentional, it's quite subtle -- please add some comments.